app/src/main/res/layout/fragment_entry.xml

```diff
@@ -16,6 +16,16 @@
     <ListView
         android:id="@+id/entry_fields"
         android:layout_width="match_parent"
-        android:layout_height="match_parent" />
+        android:layout_height="wrap_content">
+
+    </ListView>
+
+    <ImageView
+        android:id="@+id/pic_viewer"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:adjustViewBounds="true"
+        android:scaleType="centerInside"
+        android:src="@android:drawable/ic_menu_camera" />
```

Comment:
Recommend loading this as the final element in the `ListView` to make it scrollable.  Otherwise we may end up with very little screen space for text fields.  (In the future, it would probably be useful to also place the audio playback as a  `ListView` entry.)

Reply:
I tried that for the same reason but it would crash

Reply:
TODO for another pr.